frontend/playground-frontend/src/Dashboard.tsx

```diff
@@ -170,16 +171,25 @@ const Overlay = () => {
   );
 };
 
-const FilledGrid = (props) => {
+type Execution = {
```

Comment:
Recommend to always use interfaces over types unless if not possible. See https://stackoverflow.com/a/65948871/11031425

Reply:
@farisdurrani, do you think we can add a message in discord about general good typescript rules since i feel like similar comments on typescript related code changes could come up and it could be a good opportunity to educate the team